src/components/postdetail/RecruitmentStatus.module.scss

```
@@ -0,0 +1,73 @@
+.header {
+    display: flex;
+    flex-direction: column;
+    margin-top: 130px;
+    margin-bottom: 15px;
+  }
+  
+  .title {
+    color: var(--Grey-Grey-900, #222);
+    font-size: 32px;
+    font-weight: 500;
+    letter-spacing: -0.32px;
+    margin-bottom: 15px;
+  }
+  
+  .divider {
+    width: 100%;
+    height: 1px;
+    background: #9e9e9e;
+  }
+  
+  .tableHeader {
+    display: grid;
+    grid-template-columns: 300px 300px 300px;
+    gap: 0px;
+    width: 100%;
+    color: var(--Grey-Grey-600, #757575);
+    font-size: 26px;
+    font-weight: 500;
+    letter-spacing: -0.26px;
+    margin-bottom: 15px;
+    padding-left: 25px;
+  }
+  
+  .column {
+    text-align: left;
+  }
+  
+  .recruitmentRow {
+    display: grid;
+    grid-template-columns: 300px 300px 300px;
+    align-items: center;
+    width: 100%;
+    height: 80px;
+    padding: 0 25px;
+    border-radius: 10px;
+    border: 1px solid var(--Grey-Grey-500, #9e9e9e);
+    margin-bottom: 15px;
+  }
+  
+  .cell {
+    display: flex;
+    align-items: center; 
+    color: var(--Grey-Grey-800, #424242);
+    font-size: 24px;
+    font-weight: 500;
+    letter-spacing: -0.24px;
+  }
+  
+  .button {
+    display: flex;
+    padding: 5px 15px;
+    justify-content: center;
+    align-items: center;
```

글로벌 스타일 mixin 파일에 정의되어있어서 더 간편하게 쓸 수 있어요!

```suggestion
   @include flex-center
```